.gitlab-ci.yml

```diff
@@ -236,6 +237,25 @@ API:
         INTERNAL_NETWORK: ["true"]
         DISTRO_CODE: ["rhel_90"]
 
+NIGHTLY_FAIL:
+  stage: slack-report
```

Comment:
Can we please move the new stages into the finish state? That stage is used for status reporting.

Reply:
Yeah sure, didn't realize I could use that instead of creating another stage :)